src/modules/home/components/LoggedInHome/__mocks__/VisitRequests.ts

```diff
@@ -12,6 +14,24 @@ export const heroLink = {
 
 export const heroImage = { src: '/images/hero.png', alt: 'hero image' };
 
+export interface HeroProps {
```

Comment:
Ik zou de interfaces in een `.types` file zetten ze sorteren voor `LoggedInHome` en `LoggedOutHome`. Idem voor de mocks.